database/seeders/IdentitySeeder.php

```diff
@@ -4,221 +4,257 @@
-            ],
-            [
-                'name' => __('Asian'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('South Asian'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Southeast Asian'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Middle Eastern'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Latin American'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('African'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Women'),
-                'clusters' => ['gender', 'gender-and-sexuality'],
-            ],
-            [
-                'name' => __('Men'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'reachable-when-mixed'],
-            ],
-            [
-                'name' => __('Non-binary people'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'gender-diverse'],
-            ],
-            [
-                'name' => __('Gender non-conforming people'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'gender-diverse'],
-            ],
-            [
-                'name' => __('Gender fluid people'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'gender-diverse'],
-            ],
-            [
-                'name' => __('First Nations'),
-                'clusters' => ['indigenous'],
-            ],
-            [
-                'name' => __('Inuit'),
-                'clusters' => ['indigenous'],
-            ],
-            [
-                'name' => __('Métis'),
-                'clusters' => ['indigenous'],
-            ],
-            [
-                'name' => __('Supporters'),
-                'clusters' => ['lived-experience', 'reachable-when-mixed'],
-            ],
-        ];
+class IdentitySeeder extends Seeder {
+    public function run() {
 
-        foreach ($identities as $identity) {
-            Identity::firstOrCreate([
-                'name' => [
-                    'en' => $identity['name'],
-                    'fr' => trans($identity['name'], [], 'fr'),
-                ],
-                'description' => [
-                    'en' => $identity['description'] ?? null,
-                    'fr' => isset($identity['description']) ? trans($identity['description'], [], 'fr') : null,
-                ],
-                'clusters' => $identity['clusters'] ?? [],
-            ]);
+        // option to use environment to restore or backup to different environment files
+        if (config('seeder.environment') !== null && in_array(config('seeder.environment'), ['production', 'staging', 'local']) === true) {
```

Comment:
The environment array should have come from the config file. Same throughout.

Reply:
Where does `config('seeder.environment')` come from. Is that generated by running the `db:seed:backup` command?

Reply:
`config('seeder.environment')` is how we pass the environment option that was set in the command to the seeder.

Reply:
> The environment array should have come from the config file. Same throughout.

This will be in the next commit.